app/src/main/java/com/duckduckgo/app/trackerdetection/model/TrackingEvent.kt

```diff
@@ -16,12 +16,10 @@
 
 package com.duckduckgo.app.trackerdetection.model
 
-import com.duckduckgo.app.global.UriString.Companion.host
-
 data class TrackingEvent(
     val documentUrl: String,
     val trackerUrl: String,
-    val trackerNetwork: TrackerNetwork?,
-    val blocked: Boolean,
-    val entity: String = trackerNetwork?.name ?: host(trackerUrl) ?: trackerUrl
+    val categories: List<String>?,
```

Comment:
is an empty list of categories the same as a null list? if so, could we make this non-nullable?

Reply:
Good suggestion, I'll make that an empty list.

Reply:
Now that I'm making the change it feels a little clunky to have an empty list here as other variables can in the type can also be null. Do you feel strongly about this? Happy to zoom if you do.

Reply:
It's not so much that i feel strongly; it's more than i'm less sure what to do with a null list than an empty one. I suspect the answer is "treat them both the same" (in which case, i'd prefer not having to push handling both cases onto all other classes that use it)

i think the list case is different than the others. with the others, it's either null or it isn't. with a list, it's either null, or empty, or populated. 3 states instead of 2.

Again, i don't feel strongly if you think this is the best representation. but it seems more confusing to me to allow a null list here.